godot_project/editor/input_handlers/molecular_structures/dialog_input_handler.gd

```diff
@@ -0,0 +1,27 @@
+extends InputHandlerBase
```

Comment:
This inut handler has the same job the `exclusive` property used to do, preventing changes on the project while dialogs are open. However it will not prevent users from using the side panel